tools/clang_tidy/lib/clang_tidy.dart

```diff
@@ -112,9 +123,16 @@ class ClangTidy {
     final List<dynamic> buildCommandsData = jsonDecode(
       options.buildCommandsPath.readAsStringSync(),
     ) as List<dynamic>;
-    final List<Command> changedFileBuildCommands = await getLintCommandsForChangedFiles(
+    final List<List<dynamic>> shardBuildCommandsData = options
```

Comment:
I know the rest of this file does this, but nit don't use `dynamic` in null safe code, use `Object?` or `Object` if you know it won't be null.

Reply:
done